tools/themis/src/types.rs

```diff
@@ -65,25 +64,21 @@ impl ComplianceError {
             c_none = style::reset()
         );
 
-        for PackageOutcome { pkg, value } in &self.outliers {
-            let Package { parsed: pkg, .. } = &**pkg;
+        for Outlier { path, found } in &self.outliers {
             report.push_str(&format!(
-                "\n \u{2022} {c_name}{}{c_none} v{} {c_path}({}){c_none}{}",
-                pkg.name,
-                pkg.version,
-                pkg.manifest_path.strip_prefix(&workspace.root).unwrap(),
-                match value {
+                "\n {c_path}\u{21b3} {}{c_none}{}",
+                path.strip_prefix(&workspace.root).unwrap(),
```

Comment:
I'd push this logic to the place where we *create* an `Outlier`, that way we could get rid of the custom report.

Reply:
I'm unconvinced this would be any better. Aside from unnecessarily duplicating that strip call across all compliance checks, it doesn't alleviate the need for the custom report. It also walks back on the static ownership requirement for `ComplianceError` when used with `bail!()`

Reply:
I mean that's the only reason why we need to pass in `workspace: &Workspace`. If we remove that, than `Display`'s signature would work, and we can do 

```
        if let Err(err) = rule(&workspace) {
            failed |= true;
            eprlintln!("{}", err.downcast::<types::ComplianceError>()?);
        }
```

Reply:
No strong opinion though!